examples/reshare/src/engine.rs

```diff
@@ -318,6 +312,7 @@ where
             mpsc::Receiver<handler::Message<Block<H, C, V>>>,
             commonware_resolver::p2p::Mailbox<handler::Request<Block<H, C, V>>>,
         ),
+        update_cb: Box<dyn UpdateCallBack<V, C::PublicKey>>,
```

Comment:
does it make sense to alias `UpdateCallBack` as `Box<dyn UpdateCallBack<V, C::PublicKey>>`?

Reply:
I don't think so, don't like hiding `Box<dyn>` behind an alias unless the type is really cumbersome.

it would maybe look like `DynUpdateCallBack<V, C::PublicKey>` but writing out `Box<dyn>` instead of `Dyn` is only an extra 5 characters, and is much clearer imo.